### PR TITLE
Equivalence

### PR DESCRIPTION
**Pull request by MatejKomlossy**

### CODE REVIEW

File: tests/TestFormula.elm [hunk 1]
```diff
@@ -231,6 +236,7 @@ parseTests =
         , test "Disj" <| \() -> testParse "(a|b)" <| Disj a b
         , test "Conj" <| \() -> testParse "(a&b)" <| Conj a b
         , test "Impl" <| \() -> testParse "(a->b)" <| Impl a b
+        , test "Equiv" <| \() -> testParse "(a↔b)" <| Equiv a b
```
Comment (yoyko) [inline tests/TestFormula.elm:249]: Also test "ascii" version (`<->`), ideally also add the utf8 `Impl` version.

Comment (crnkjck) [inline tests/TestFormula.elm:239]: Add version with '<->'

File: tests/TestFormula.elm [hunk 2]
```diff
@@ -182,9 +183,11 @@ isSubformulaOfTests =
         , describe "Conj" <| binIsSubformulaTests Conj a b
         , describe "Disj" <| binIsSubformulaTests Conj a b
         , describe "Impl" <| binIsSubformulaTests Conj a b
+        , describe "Equiv" <| binIsSubformulaTests Conj a b
```
Comment (crnkjck) [inline tests/TestFormula.elm]: Fix these to really test the respective connective, e.g, `binIsSubformulaTests Impl a b`.
Fix `binIsSubformulaTests` to really use the connective from the first argument instead of `Conj`.

File: tests/TestFormula.elm [hunk 3]
```diff
@@ -231,6 +236,7 @@ parseTests =
         , test "Disj" <| \() -> testParse "(a|b)" <| Disj a b
         , test "Conj" <| \() -> testParse "(a&b)" <| Conj a b
         , test "Impl" <| \() -> testParse "(a->b)" <| Impl a b
```
Comment (crnkjck) [inline tests/TestFormula.elm]: Add tests for unicode versions and all variations of all symbols.

**Review by crnkjck — CHANGES_REQUESTED** [review]
Some legacy issues should be fixed.

**Review by crnkjck — APPROVED** [review]
